pkg/services/stats/statsimpl/stats.go

```diff
@@ -74,7 +74,9 @@ func (ss *sqlStatsService) getTagCount(ctx context.Context, orgs []*org.OrgDTO)
 
 func (ss *sqlStatsService) getFolderCount(ctx context.Context, orgs []*org.OrgDTO) (int64, error) {
 	total := 0
+	ctx = identity.WithBackgroundCallFlag(ctx)
 	for _, org := range orgs {
+		// TODO: How to replace this?
```

Comment:
@gamab @kalleep Do you have any idea? Should we just create a new `authn.Claims[authn.AccessTokenClaims]` and set the namespace to the org?

Reply:
So I don't think you need to do `ctx = identity.WithRequester(ctx, backgroundUser)` but you need to pass that "fake" user in the query, Otherwise the service will error out :(